problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,56 @@
+const MyQueue = function () {
+  this.helperStack = [];
+  this.stackWorkAsQueue = [];
+};
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.helperStack.push(x);
+};
+
+/**
+ * leetcode problem link: https://leetcode.com/problems/implement-queue-using-stacks
+ *
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  const length = this.helperStack.length;
+  if (!this.stackWorkAsQueue.length) {
+    for (let i = 0; i < length; i++) {
+      const num = this.helperStack.pop();
+      this.stackWorkAsQueue.push(num);
+    }
+    return this.stackWorkAsQueue.pop();
+  } else {
+    return this.stackWorkAsQueue.pop();
+  }
+};
```

Comment:
```markdown
1. helperStack : 들어오는 값을 누적합니다.
2. stackWorkAsQueue : 선입선출 구현을 위해 값을 이동시킬 때 사용합니다.

stackWorkAsQueue가 비어있다면, helperStack의 값을 이동시키고 아니라면 바로 stackWorkAsQueue의 마지막 값을 반환합니다.
```

위와 같은 로직이라고 이해하였습니다! 여기서 stackWorkAsQueue의 동작이 반복되니, 아래와 같이 수정할 수 있지 않을까요?

```suggestion
MyQueue.prototype.pop = function () {
  const length = this.helperStack.length;
  
  if (!this.stackWorkAsQueue.length) {
    for (let i = 0; i < length; i++) {
      this.stackWorkAsQueue.push(this.helperStack.pop());
    }
  }
    
  return this.stackWorkAsQueue.pop();
};
```